### DISCUSSION TITLE
Welcome to VidLii Discussions!

### DISCUSSION THREAD

**Discussion by lifesminder** [lifesminder/vidlii#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tchthgh** [comment]
hi

**Reply by lifesminder** [reply]
hi!

**Reply by tchthgh** [reply]
yo, what are you doing

**Reply by lifesminder** [reply]
> yo, what are you doing

hey, nothing much besides personal stuff fix, wbu?

**Reply by tchthgh** [reply]
doing stuff, that's all
